sw/device/silicon_creator/lib/drivers/flash_ctrl.c

```diff
@@ -278,10 +281,31 @@ static void page_lockdown(flash_ctrl_info_page_t info_page) {
 void flash_ctrl_init(void) {
   // Initialize the flash controller.
   abs_mmio_write32(kBase + FLASH_CTRL_INIT_REG_OFFSET,
-                   bitfield_bit32_write(0u, FLASH_CTRL_INIT_VAL_BIT, true));
+                   bitfield_bit32_write(0, FLASH_CTRL_INIT_VAL_BIT, true));
   // Disable all access to the silicon creator secret info page.
   page_lockdown(kFlashCtrlInfoPageCreatorSecret);
   sec_mmio_write_increment(2);
+  // Configure default scrambling, ECC, and HE settings for the data partition.
```

Comment:
This should work. During manufacturing, we can use a program to configure the OTP settings, and then trigger a second bootstrap to load the data partition. We should document how this would work for the info pages in question, and specify if there are any additional provisioning requirements for enabling the mask ROM.